softsmithy-lib-core/src/main/java/org/softsmithy/lib/text/Formatter.java

```diff
@@ -1,25 +1,25 @@
-/*
- *         COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Notice
- *
- * The contents of this file are subject to the COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL)
- * Version 1.0 (the "License"); you may not use this file except in
- * compliance with the License. A copy of the License is available at
- * http://www.opensource.org/licenses/cddl1.txt
- *
- * The Original Code is SoftSmithy Utility Library. The Initial Developer of the
- * Original Code is Florian Brunner (Sourceforge.net user: puce). All Rights Reserved.
- *
- * Contributor(s): .
- */
-package org.softsmithy.lib.text;
-
-/**
- *
- * @author puce
- */
-public interface Formatter<T> {
-
-    String format(T t) throws FormatException;
-
-    void format(T t, Appendable appendable) throws FormatException;
-}
+/*
+ *         COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Notice
+ *
+ * The contents of this file are subject to the COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL)
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.opensource.org/licenses/cddl1.txt
+ *
+ * The Original Code is SoftSmithy Utility Library. The Initial Developer of the
+ * Original Code is Florian Brunner (Sourceforge.net user: puce). All Rights Reserved.
+ *
+ * Contributor(s): .
+ */
+package org.softsmithy.lib.text;
+
+/**
+ *
+ * @author puce
+ */
+public interface Formatter<T> {
+
+    String format(T t) throws FormatException;
+
+    Appendable format(T t, Appendable appendable) throws FormatException;

```

Comment:
Add Javadoc
State that the return value is the same as the input and intended to allow fluent coding...